instructor/ui/instructor.js

```diff
@@ -107,12 +107,14 @@ $(function(){
             );
             $('.jqDialog_header').remove();
             $('#jqDialog_box').prepend("<h1 class='jqDialog_header' role='heading' >Warning</h1>");
+            showed_warning=true;
```

Comment:
As a user, would you have expected to continue showing the message if you clicked 'no'?